CMakeLists.txt

```diff
@@ -516,7 +516,9 @@ if(BUILD_SHARED_LIBS)
       DESTINATION python/El/core)
   endif()
 endif()
-
+if(not NO_BINARY_SUBDIRECTORIES)
```

Comment:
The capitalization of 'not' here seems to be inconsistent with the rest of the change set.